package.json

```diff
@@ -6,7 +6,11 @@
   "main": "run.js",
   "scripts": {
     "test": "NODE_PATH=./node_modules jest",
-    "changelog": "conventional-changelog -p angular -i CHANGELOG.md -s"
+    "changelog": "conventional-changelog -p angular -i CHANGELOG.md -s",
+    "lint": "eslint --ignore-path .gitignore ."
+  },
+  "engines": {
+    "node": ">=12.0.0"
```

Comment:
This should be a separate change IMO and by that a separate entry in the changelog.
I would not necessary consider it breaking as v12 is the oldest active release and v10 will be out of maintenance around May 2021 as shown on the [Node.js Releases](https://nodejs.org/en/about/releases/).

Reply:
I'm not changing the version that this action uses. I just added the engine to `package.json` so that eslint can know that I'm using node 12 and allow features present in this version, such as rest/spread. As you can see [here](https://github.com/wagoid/commitlint-github-action/blob/style/add-eslint-config/Dockerfile#L1), the action was already using node v12 so nothing will change for its users :smiley:

Reply:
Okay, I see.